lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -289,8 +289,8 @@ function _dosomething_campaign_pitch_page_access($node) {
     if (dosomething_campaign_get_campaign_type($node) == 'sms_game') {
       return FALSE;
     }
-    // Is the user a DS admin or editor?
-    if (dosomething_user_is_staff()) {
+    // Is the user a DS admin, regional admin or editor?
```

Comment:
small thing, just update comment to say is the user on DS staff, or regional admin